mysql-test/suite/tokudb.parts/r/partition_auto_increment_tokudb.result

```diff
@@ -1600,14 +1390,13 @@ t1	CREATE TABLE `t1` (
   `a` bigint(20) unsigned NOT NULL AUTO_INCREMENT,
   `b` varchar(64) DEFAULT NULL,
   PRIMARY KEY (`a`)
-) ENGINE=TokuDB AUTO_INCREMENT=18446744073709551615 DEFAULT CHARSET=latin1
-/*!50100 PARTITION BY HASH (a)
+) ENGINE=TokuDB AUTO_INCREMENT=18446744073709551615 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci
+/*!50100 PARTITION BY HASH (`a`)
 PARTITIONS 3 */
-Warnings:
-Warning	1287	The partition engine, used by table 'test.t1', is deprecated and will be removed in a future release. Please use native partitioning instead.
 SELECT * FROM t1;
 a	b
 18446744073709551613	2
 5	non-generated number
 DROP TABLE t1;
+SET SESSION information_schema_stats_expiry=default;
```

Comment:
nit-pick here but it isn't necessary to reset session variables

Reply:
The test includes common .inc file. I think this is a try to restore the state after the *.inc file execution inside of one *.test file.